config/locales/views/settings/en.yml

```
@@ -1,554 +1,555 @@
-        query_id_placeholder_new: Enter your IBKR Flex Query ID
-        query_id_placeholder_existing: Leave blank to keep the current Query ID
-        token_label: Token
-        token_placeholder_new: Enter your IBKR Flex Web Service Token
-        token_placeholder_existing: Leave blank to keep the current Token
-        save_configuration: Save Configuration
-        update_configuration: Update Configuration
-        status_configured_prefix: "%{summary}. Visit the"
-        accounts_tab: Accounts
-        status_configured_suffix: tab to manage discovered accounts.
-        not_configured: Not configured.
+---
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)